src/neptune/api/searching_entries.py

```diff
@@ -97,7 +98,8 @@ def get_single_page(
     types: Optional[Iterable[str]],
     query: Optional["NQLQuery"],
     searching_after: Optional[str],
-) -> Any:
+    use_proto: Optional[bool] = None,
```

Comment:
Why is this optional? Cannot be just `use_proto: bool = False`?

Reply:
`None` means -> keep what environment variable says. `True`/`False` will override it. Used for testing purposes.